FirstProject/new-evaluation.html

```diff
@@ -517,11 +524,13 @@ <h2 class="description_section" style="color: #03A9F4;">Workflow, Leadership & S
                     </li>
                 </ul>
             </fieldset> <br>
-            <input class="submit_button" type="submit" value="SUBMIT" style="float: right; margin-top: 30px; padding: 15px 40px; margin-right: 100px;">
+            <div class="#" style="width: 73%; margin: 0 auto;">
```

Comment:
I just wanted to style that button. I could add any value to the class. That was just a random choice. :)
Should I avoid it in the future?
Do you have any suggestion for me?